sdcm/nemesis.py

```diff
@@ -470,6 +485,13 @@ def disrupt_multiple_hard_reboot_node(self):  # pylint: disable=invalid-name
                 self.log.info('Waiting JMX services to start after node reboot')
                 self.target_node.wait_jmx_up()
             self.cluster.wait_for_nodes_up_and_normal(nodes=[self.target_node])
+            found_cdc_error = list(cdc_expected_error)
+            found_success_info = list(cdc_success_msg)
+            if found_cdc_error and not found_success_info:
+                raise CdcStreamsWasNotUpdated(
+                    f"After '{found_cdc_error[0]}', none of messages '{' or '.join(cdc_success_msg)}'")
```

Comment:
i'm not sure i understand what you are trying to do here

Reply:
@aleksbykov , could you please put some explanation into the comments

Reply:
Done